src/nl/hannahsten/texifyidea/action/analysis/WordCountAction.kt

```diff
@@ -127,17 +138,23 @@ open class WordCountAction : AnAction(
         val bibliography = bibliographies.flatMap { it.childrenOfType(LatexNormalText::class) }
 
         val (wordsNormal, charsNormal) = countWords(allNormalText)
+        val (wordsParameter, charsParameter) = countWords(parameterText)
         val (wordsBib, charsBib) = countWords(bibliography)
 
-        return Pair(wordsNormal - wordsBib, charsNormal - charsBib)
+        return Pair(wordsNormal + wordsParameter - wordsBib, charsNormal + charsParameter - charsBib)
     }
 
+    private val LatexParameterText.command: PsiElement?
+        get() {
+            return this.parent?.parent?.parent?.prevSibling
+        }
```

Comment:
Please place values above functions and below the class header as per kotlin conventions.

This would also probably be more suited for a utility extension function in `nl.hannahsten.texify.util.PsiKt` or `nl.hannahsten.texify.util.CommandsKt`.

Also, this can be implemented more generally with something like `.parentOfType(LatexCommands::class).commandToken`  (or even a newly defined `enclosedCommand()`) That way you're less dependent on the exact psi tree structure.